src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs

```diff
@@ -160,10 +165,9 @@ internal void SetForeground(object localValue)
 					break;
 				case GradientBrush gradient:
 					this.SetStyle(
-						("background", gradient.ToCssString(this.RenderSize)),
-						("color", "transparent"),
-						("background-clip", "text")
```

Comment:
The deleted `background-clip` here is because we already define that in Uno.UI.css